src/shared/functions.py

```diff
@@ -224,12 +224,9 @@ def get_s3_keys(
     return contents
 
 
-def get_s3_site_filename_suffix(s3_path: str):
-    """Extracts site/filename data from s3 path"""
-    # The expected s3 path for site data packages looks like:
-    #   s3://bucket_name/enum_value/site/study/data_package/file
-    # so this is returning data_package/file
-    return "/".join(s3_path.split("/")[-3:])
+def get_s3_filename(s3_path: str):
+    """ Given an s3 path/key, returns the filename"""    
```

Comment:
nit: being slightly pedantic, you could probably just call this `get_filename` (since there's nothing S3 about it) - but this is also fine.

Reply:
`get_filename_from_s3_path` is probably the more explicit function name